projects/make-a-vis/src/lib/visualization-view/main/main.component.scss

```diff
@@ -58,8 +58,13 @@
 }
 
 .visualization-container {
-    height: calc(93vh - 1vh - 285px); /* .main-view is 93vh with 1vh bottom padding in .visualization-view  and .legend is 200px at top-level, 70px - assumed padding(TODO - FIX needed) */
+    height: calc(93vh - 1vh - 285px - 48px); /* .main-view is 93vh with 1vh bottom padding in .visualization-view  and .legend is 200px at top-level, 70px - assumed padding(TODO - FIX needed) */
```

Comment:
Work with Leonard to determine how to overlay the expanded about section without moving the legend display down. I think the option of using a `mat-menu` might do the trick.

Reply:
used mat-menu instead of expansion panel.